tools/driver/p4c_src/p4c.bmv2.cfg

```diff
@@ -28,7 +28,7 @@ class BMV2Backend(BackendDriver):
         self.enable_commands(['preprocessor', 'compiler'])
 
         # options
-        self.add_command_option('preprocessor', "-E -x c")
+        self.add_command_option('preprocessor', "-E -x assembler-with-cpp")
```

Comment:
It looks like `-undef` and `-nostdinc` appear in tools/driver/p4c_src/driver.py -- perhaps the `-x assembler-with-cpp` should be there instead of here in the .cfg files?